resources/setup_emulator.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e
+
+kill_all_emus() {
+    for emu_device in $(adb devices -l |grep 'device product:' |cut -d' ' -f1); do
+        echo "Killing emulator: $emu_device"
+        adb -s $emu_device emu kill
+    done
+}
+
+boot_finished_status="stopped"
+attempt=0
+timeout=300
```

Comment:
I would recommend standardizing variable names to use capital letters

Reply:
done